cluster-scope/base/operatorgroups/cluster-logging-operator/operatorgroup.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
```

Comment:
You don't need to specify a namespace in here, since the `namespace` from `kustomization.yaml` would overwrite this anyways. You can delete those from multiple resources in here.

Reply:
fixed here, and multiple other places :-)